services/booking/search.rb

```diff
@@ -0,0 +1,16 @@
+class BookingService
+  class Search < ApplicationService
+    def initialize(params)
+      @route_id = params[:route_id]
+    end
+
+    def call
+      orders = Order.where(route: @route_id)
```

Comment:
`attr_reader` + private, avoid directly using instance variables
Also, move orders definition to a separate method /w memoization